src/main/java/com/kelinci/mainapp/api/UserController.java

```diff
@@ -6,27 +6,49 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
 @RestController
 public class UserController {
     private OurUser registeredUser;
     private OurUser confirmedUser;
 
+    private List<OurUser> listOfRegisteredUsers = new ArrayList<>();
```

Comment:
Do zastanowienia: czy potrzebujemy dwóch list? Wszak obiekt OurUser ma flagę isConfirmed. To jakby się zastanowić, to pewnie dałoby się wykorzystać jedną listę.